src/sys/wait.rs

```diff
@@ -184,6 +191,12 @@ fn decode(pid : pid_t, status: i32) -> WaitStatus {
     } else if status::signaled(status) {
         WaitStatus::Signaled(pid, status::term_signal(status), status::dumped_core(status))
     } else if status::stopped(status) {
+        #[cfg(any(target_os = "linux", target_os = "android"))] {
```

Comment:
Oh yeah, should have pointed out where that comes from. We already have it in nix, so it's not a risk to use it.